geomap/src/components/LeafletMap.vue

```diff
@@ -56,24 +59,67 @@ import {
 import MarkerCluster from "./MarkerCluster.vue";
 import { Vue } from "vue-class-component";
 import { useStore } from "vuex";
+import L from "leaflet";
+import { Alarm, Node } from "@/types";
```

Comment:
Couldn't these two just get added to the import below it?

Reply:
Removed extra import.